adapter/exporter/src/main/java/org/mobilitydata/gtfsvalidator/exporter/ProtobufNoticeExporter.java

```diff
@@ -773,6 +773,22 @@ public void export(final StopTimeArrivalTimeAfterDepartureTimeNotice toExport) t
                 .writeTo(streamGenerator.getStream());
     }
 
+    @SuppressWarnings("unchecked")
+    @Override
+    public void export(final FastTravelBetweenStopsNotice toExport) throws IOException {
+        protoBuilder.clear()
+                .setCsvFileName(toExport.getFilename())
+                .setType(((List<Integer>) toExport.getNoticeSpecific(KEY_COMPOSITE_KEY_SECOND_VALUE)).size() > 2 ?
+                        GtfsValidationOutputProto.GtfsProblem.Type.TYPE_FAST_TRAVEL_BETWEEN_FAR_STOPS :
+                        GtfsValidationOutputProto.GtfsProblem.Type.TYPE_FAST_TRAVEL_BETWEEN_CONSECUTIVE_STOPS
+                )
+                .setSeverity(GtfsValidationOutputProto.GtfsProblem.Severity.ERROR)
+                .setEntityId((String) toExport.getNoticeSpecific(KEY_COMPOSITE_KEY_FIRST_VALUE))
```

Comment:
Should we extract `KEY_COMPOSITE_KEY_FIRST_PART` and `KEY_COMPOSITE_KEY_SECOND_PART` here, or does it come as duplicate information?